packages/now-next/src/index.ts

```diff
@@ -1540,7 +1544,7 @@ export const build = async ({
 
                   if (!currentPage) {
                     console.error(
-                      "Failed to find matching page for", toRender, "in lambda"
+                      "Failed to find matching page for", {toRender, header: req.headers['x-nextjs-page'], url: req.url, pages: Object.keys(pages) }, "in lambda"
```

Comment:
Logging all page keys might get big, right? We have a max chars per log so maybe this should be a separate console.error()?